src/Entity/Log.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+namespace EMS\CommonBundle\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\Doctrine\UuidGenerator;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @ORM\Table(name="log")
```

Comment:
Not sure but is log not a reserved keywoard in sql context? Maybe rename table name and entity to LogMessage

Reply:
Fixed